gramex/config.py

```diff
@@ -840,3 +844,78 @@ def used_kwargs(method, kwargs, ignore_keywords=False):
             target = used if key in set(argspec.args) else rest
             target[key] = val
     return used, rest
+
+
+esl_paths = AttrDict()
+esl_paths['source'] = Path(__file__).absolute().parent
+esl_conf = (+PathConfig(esl_paths['source'] / 'gramex.yaml')).get('eslog', AttrDict())
```

Comment:
Move this into `gramex/services/__init__.py` under `def eslog(conf)`. You'll get the config directly